src/librbd/Operations.h

```diff
@@ -24,6 +24,10 @@ class Operations {
 public:
   Operations(ImageCtxT &image_ctx);
 
+  uint64_t reserve_async_request_id() {
+    return m_async_request_seq++;
```

Comment:
Nit: keep as prefix increment to match other ops

Reply:
Indeed. Thanks!  updated.